apis/flinkcluster/v1beta1/flinkcluster_types.go

```diff
@@ -195,6 +195,9 @@ type JobManagerSpec struct {
 	// Currently `VPC, External` are only available for GKE.
 	AccessScope string `json:"accessScope,omitempty"`
 
+	// _(Optional)_ Define JobManager Service annotations for configuration.
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Yeah I think we can add support for this. Since we have something similar for pods and the property is `PodAnnotations` I think we should have this as `ServiceAnnotations`? wdyt?

Reply:
Awesome, I agree, this would be more consistent
I'll implement changes as soon as possible

Reply:
@regadas Updated, also refreshed branch